core/node/base_token_adjuster/src/base_token_ratio_persister.rs

```diff
@@ -90,53 +83,63 @@ impl BaseTokenRatioPersister {
         let new_ratio = self.retry_fetch_ratio().await?;
         self.persist_ratio(new_ratio).await?;
 
-        let max_attempts = self.config.l1_tx_sending_max_attempts;
-        let sleep_duration = self.config.l1_tx_sending_sleep_duration();
-        let mut result: anyhow::Result<()> = Ok(());
-        let mut prev_base_fee_per_gas: Option<u64> = None;
-        let mut prev_priority_fee_per_gas: Option<u64> = None;
+        if let Some(l1_params) = &self.l1_params {
```

Comment:
nit:
`let Some(l1_params) = &self.l1_params else { return Ok(()) }`
will look nicely

Reply:
It's much nicer indeed - fixed